.github/workflows/go.yml

```diff
@@ -1,66 +1,118 @@
+        id: go
 
-            - name: Check out code into the Go module directory
-              uses: actions/checkout@v2
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
 
-            - name: Get dependencies
-              run: |
-                  go get -v -t -d ./...
-                  if [ -f Gopkg.toml ]; then
-                      curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-                      dep ensure
-                  fi
-            - name: Build
-              run: go build -v .
-    test:
-        name: Test
-        runs-on: ubuntu-latest
-        steps:
-            - name: Check out code into the Go module directory
-              uses: actions/checkout@v2
-            - name: Get dependencies
-              run: |
-                  go get -v -t -d ./...
-                  if [ -f Gopkg.toml ]; then
-                      curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-                      dep ensure
-                  fi
-            - name: Test (Base)
-              run: go test
-            - name: Test (nativestore)
-              run: go test ./nativestore
-            - name: Test (lib)
-              run: go test ./lib
-            - name: Test (node)
-              run: go test ./node
-            - name: Test (vscodesupport)
-              run: go test ./vscodesupport
-            - name: Test (gh)
-              run: go test ./gh
-            - name: Test (git)
-              run: go test ./git
-            - name: Setup Miniconda
-              uses: goanpeca/setup-miniconda@v1.3.0
-            - name: Test (python)
-              run: go test ./python
-
-              
-            - name: Build
-              run: go build -v .
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+          if [ -f Gopkg.toml ]; then
+              curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+              dep ensure
+          fi
+      - name: Build
+        run: go build -v .
+  test-linux:
+    name: Test (Linux)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+          if [ -f Gopkg.toml ]; then
+              curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+              dep ensure
+          fi
+      - name: Test (Base)
+        run: go test
+      - name: Test (nativestore)
+        run: go test ./nativestore
+      - name: Test (lib)
+        run: go test ./lib
+      - name: Test (node)
+        run: go test ./node
+      - name: Test (vscodesupport)
+        run: go test ./vscodesupport
+      - name: Test (gh)
+        run: go test ./gh
+      - name: Test (git)
+        run: go test ./git
+      - name: Test (finder)
+        run: go test ./finder
```

Comment:
Oh wait I just realized that these tests leave out the python tests. Is there a reason for this? Or can we add them back?

Reply:
I actually don't know why that is... Let me pull from master and see if it updates.